pkgs/development/python-modules/keymap-drawer/tests/default.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  fetchFromGitHub,
+  runCommand,
+  stdenv,
+  testers,
+
+  keymap-drawer,
+  yamllint,
+}:
+let
+  runKeymapDrawer =
+    name:
+    runCommand "keymap-drawer-${name}" {
+      nativeBuildInputs = [ keymap-drawer ];
+    };
+
+  MattSturgeon-example = fetchFromGitHub {
+    owner = "MattSturgeon";
+    repo = "glove80-config";
+    rev = "d55267dd26593037256b35a5d6ebba0f75541da5";
+    hash = "sha256-MV6cNpgHBuaGvpu2aR1aBNMpwPnDqOSbGf+2ykxocP4=";
+    nonConeMode = true;
+    sparseCheckout = [
+      "config"
+      "img"
+    ];
+  };
+
+  # MattSturgeon's example requires MDI icons
+  mdi = fetchFromGitHub {
+    owner = "Templarian";
+    repo = "MaterialDesign-SVG";
+    tag = "v7.4.47";
+    hash = "sha256-NoSSRT1ID38MT70IZ+7h/gMVCNsjNs3A2RX6ePGwuQ0=";
+  };
+in
+{
+  dump-config = runKeymapDrawer "dump-config" ''
+    keymap dump-config --output "$out"
+
+    if [ ! -s "$out" ]; then
+      >&2 echo 'Expected `dump-config` to have content.'
```

Comment:
The redirection-before-command syntax (>&2 echo) is valid but uncommon.
For readability and consistency, consider moving the redirection to the end.

Reply:
I can see where you're coming from.

Ironically, I tend to write "redirect to stderr" this way because I find it _more_ readable! My logic is that it's easily noticed when it prefixes the command, while it is easy to miss when at the end of a long line.

As for consistency, I found 34 matches grepping for `^ *>&2` treewide. I think I actually learnt this pattern _from_ nixpkgs 😅 

I was hoping something like `pkgs/stdenv/generic/setup.sh` may have a "writeErr"-style function that could be used for even better readability, but it doesn't look like it.